### PR TITLE
Release notes for Client libraries for Preview 1

### PR DESCRIPTION
**Pull request by maririos**

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj [hunk 1]
```diff
@@ -6,8 +6,7 @@
     <VersionPrefix>4.0.0</VersionPrefix>
     <PackageTags>Microsoft Azure Key Vault Keys</PackageTags>
     <PackageReleaseNotes>
-      <![CDATA[
-      ]]>
+      For release notes and more information please visit https://aka.ms/azure-sdk-preview1-net
```
Comment (AlexGhiondea) [inline sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj]: Could you double check that the package will correctly have this in it? Other than that, LGTM!

**Review by AlexGhiondea — APPROVED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj (hunk 1 above)
Comment (maririos) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj]: The link currently doesn't work. Is that expected?

Comment (AlexGhiondea) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj]: Yes -- we will update that shortly.

Comment (maririos) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj]: VS doesn't show it (they have a bug), but looking at the `nuspec `we have
```
<description>This is the Microsoft Azure Key Vault Secrets client library</description>
<releaseNotes>For release notes and more information please visit https://aka.ms/azure-sdk-preview1-net</releaseNotes>
```

So it looks good

**Review by tg-msft — APPROVED** [review]
Do we need this for the other Certs.csproj?  Otherwise looks good to me.

**Comment by maririos** [comment]
We are not going to ship Certs as part of Preview 1, so I didn't bother to change it...

**Pull request merged**